Dockerfile_ams_clearlinux

```diff
@@ -47,6 +47,24 @@ RUN cmake -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE='/usr/bin/python3.8' -DPYTHON_L
 
 RUN make --jobs=$(nproc --all)
 
+FROM clearlinux:latest as opencl
+
+RUN swupd update --no-boot-update \
+    && swupd bundle-add wget unzip lftp package-utils cpio \
+    && rm -rf /var/lib/swupd
+
+RUN if [ ! -e "/usr/lib64/libzstd.so.1"  ]; then ln -s /usr/lib64/haswell/libzstd.so.1.4.5 /usr/lib64/libzstd.so.1 ; fi
+
+WORKDIR /pkg
+
+RUN mkdir -p rpms/runtime && pwd && source /etc/os-release \
```

Comment:
Just a question:
```
&& pwd &&
```
- why we need this part?

Reply:
that was just for debugging, it could be dropped